cli.js

```diff
@@ -51,18 +50,15 @@ const openGitHub = async name => {
 			return;
 		}
 
-		let url = githubUrlFromGit(repository.url);
+		const {url = packageData.homepage, warnings} = repoUrlFromPackage(packageData);
 
-		if (!url) {
-			url = repository.url;
-
-			if (isUrl(url) && /^https?:\/\//.test(url)) {
-				console.error(`${logSymbols.error} The \`repository\` field in package.json should point to a Git repo and not a website. Please open an issue or pull request on \`${name}\`.`);
-			} else {
-				console.error(`${logSymbols.error} The \`repository\` field in package.json is invalid. Please open an issue or pull request on \`${name}\`. Using the \`homepage\` field instead.`);
+		for (const warning of warnings) {
+			console.error(`${logSymbols.error} ${warning}`);
+		}
 
-				url = packageData.homepage;
-			}
+		if (!url) {
+			console.error(`${logSymbols.error} No \`homepage\` field found in package.json.`);
```

Comment:
I think this message should also mention `repository` field, because this warning is about the package missing both. And also include the "Please open an issue or pull request on" part.